tests/diff.yml

```diff
@@ -5,22 +5,34 @@ args: ["-timeout=30s", "-shuffle=on", "-race", "./..."]
 results:
   ferretdb:
     stats:
-      expected_fail: 49
-      expected_pass: 20
+      expected_fail: 53
+      expected_pass: 22
+      expected_skip: 2
 
     pass:
       - regex: FerretDB$
 
     fail:
       - not_regex: FerretDB$
 
+    # https://github.com/FerretDB/dance/issues/266
+    skip:
+      - github.com/FerretDB/dance/tests/diff/TestFloatValues/FindAndModify
+      - github.com/FerretDB/dance/tests/diff/TestFloatValues/Update
```

Comment:
@AlekSi this configuration doesn't seem to be correct (as dance fails), but I don't understand why. What did I do wrong?